website/docs/reference/manifest.md

```diff
@@ -41,6 +41,20 @@ Scarb bakes in the concept of [Semantic Versioning](https://semver.org/), so mak
    impls or anything else.
 4. Use version numbers with three numeric parts such as 1.0.0 rather than 1.0.
 
+### `edition`
+
+The edition key is an optional key that affects which Cairo edition your package is compiled with.
+The editions allow newer cairo compiler versions to introduce opt-in features that may break existing code.
```

Comment:
This nitpicking is crazy but as Marek mentioned on Slack, every `Cairo` mention should be uppercase I guess?